docs/examples/utilities/fixed-width-container.html

```diff
@@ -0,0 +1,35 @@
+---
+layout: examples
+title: Fixed width container
+category: _utilities
+---
+
+<div class="u-fixed-width">
+  Content inside a container with the class `u-fixed-width` matches the width and position of content inside a grid.
+</div>
+
+<script>
```

Comment:
This shouldn't be needed. Every example page has a grid switch built into layout.

Reply:
Good catch, I duplicated another example and forgot to delete this. Deleted now.